pkg/util/execdetails/execdetails.go

```diff
@@ -555,6 +555,7 @@ type basicCopRuntimeStats struct {
 	procTimes  Percentile[Duration]
 	// executor extra infos
 	tiflashScanContext TiFlashScanContext
+	tiFlashWaitSummary TiFlashWaitSummary
```

Comment:
Maybe we should use `tiflashScanContext/tiflashWaitSummary` or `tiFlashScanContext/tiFlashWaitSummary`

Reply:
Good catch， will change it.